package.json

```diff
@@ -27,10 +27,10 @@
     "postcss-custom-media": "^5.0.0",
     "postcss-custom-properties": "^5.0.0",
     "postcss-easy-import": "^1.0.1",
-    "postcss-import": "^8.0.2",
```

Comment:
@simonsmith before I merge this branche I just wanted to make sure that this was unused